Styling cards showpage

minami-77 opened the pull request:
Delete Edit button of Activities, sorry...!
![image](https://github.com/user-attachments/assets/396ae73c-aee5-47aa-9fa4-2c3efc8740f4)

Munkleson commented:
Hey Minami, sorry but could you please describe a bit more in the PR what was done in this branch?

minami-77 commented:
Sorry!
This is what I did in this branch:

- Implemented a complete/not complete checkmark on task cards without using the edit button (modal).
(Added complete/incomplete methods in tasks_controller and updated routes with patch :complete/incomplete).

- Implemented a modal to show details on activity cards without an "edit" option (The code for editing activities is currently commented out).

- Fixed the issue where saved tasks from the preview page were not saving the correct text.
(Modified the save_event_plan method in events_controller to ensure task suggestions are properly parsed as JSON/Hash)

Munkleson commented:
Thank you for that! Merging
